com.playeveryware.eos/Runtime/Core/EOS_SDK_Additions/PlayerDataStorageFileTransferRequestWrapper.cs

```diff
@@ -53,7 +53,8 @@ public override Result CancelRequest()
 
         public override void Release()
         {
-            _instance.Release();
+            _instance?.Release();
+            _instance = null;
```

Comment:
I believe that you should rebase / merge `development` into this branch and this change will go away.

Reply:
Merged it in, then rebased. This is out of the PR now.